### PR TITLE
599 dlellipsis does not support multi line ellipsis

### PR DESCRIPTION
**Pull request by fadiDL**
**Thank you for your contribution to the repo. Before submitting this PR, please make sure:**
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests
- [x] You have updated documentation
- [x] You have tested your changes

**Please provide a description of the changes proposed in the pull request and reference any related issues in the repository.**

**Please indicate if this PR contains:**
- [ ] A bug fix
- [ ] A feature request
- [ ] Breaking changes
- [x] An enhancement

### CODE REVIEW

File: src/components/essential/DlEllipsis/DlEllipsis.vue [hunk 1]
```diff
@@ -11,14 +23,14 @@
...
-            v-if="rightText"
+            v-if="!multiline && rightText"
             class="dl-ellipsis__right"
         >
             {{ rightText }}
         </span>
         <dl-tooltip
             v-if="shouldShowTooltip"
-            :max-width="'max-content'"
+            :max-width="'30vw'"
```
Comment (RotemDl) [inline src/components/essential/DlEllipsis/DlEllipsis.vue:33]: Seems arbitrary - why the change?

File: src/components/essential/DlEllipsis/DlEllipsis.vue [hunk 2]
```diff
@@ -79,6 +97,21 @@ export default defineComponent({
...
+         */
+        multiline: {
+            type: Boolean,
+            default: false
+        },
+        /**
+         * Number of lines to display
+         * must be used with multiline
+         */
+        linesDisplayed: {
```
Comment (RotemDl) [inline src/components/essential/DlEllipsis/DlEllipsis.vue:112]: maxLines

**Review by RotemDl — CHANGES_REQUESTED** [review]

File: tests/DlEllipsis.spec.ts [hunk 3]
```diff
@@ -11,7 +11,9 @@ const props = {
     splitPosition: 0.5,
     tooltip: true,
     tooltipPosition: 'top middle',
-    tooltipOffset: [0, 25]
+    tooltipOffset: [0, 25],
+    multiline: false,
```
[the remark this answers is not in the document]
Comment (fadiDL) [reply tests/DlEllipsis.spec.ts:15]: Required for the test to validate the props
